### ISSUE TITLE
[CHORE] Scheduler 별 코드 수정

### ISSUE THREAD

**Issue by seoin0110** [boostcampaitech5/level1_imageclassification-cv-02#35]
## Background
- 실행 시 스케줄러 별로 오류가 발생하지 않도록 코드 수정합니다.
    
## Todo
- [x] ReduceLROnPlateau에서 scheduler.step 수정
- [x] CyclicLR에서 cycle_momentum=False로 수정
